models.py

```diff
@@ -1,6 +1,6 @@
 from datetime import datetime
 
-from room import db
+from roomutils import db
```

Comment:
Let's work on keeping this as `room` instead of `roomutils`. This powered the main application by name.